Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.py

```diff
@@ -309,24 +316,81 @@ def test_function():
         return return_error(str(e))
 
 
+def fetch_incidents(last_run: dict = None):
+    dParams = demisto.params()
+    # Get the last fetch time, if exists
+    last_fetch = last_run.get('time') if isinstance(last_run, dict) else None
+
+    # Handle first time fetch, fetch incidents retroactively
+    if last_fetch is None:
+        today = datetime.today()
+        delta = today - timedelta(days=50)
+        last_fetch = date_to_timestamp(delta)
+
+    incidents: list = []
+    dArgs = {}
+    dArgs['roleArn'] = dParams.get('roleArn')
+    dArgs['region'] = dParams.get('region')
+    dArgs['roleSessionName'] = dParams.get('roleSessionName')
+    dArgs['roleSessionDuration'] = dParams.get('roleSessionDuration')
+    dArgs['analyzerArn'] = dParams.get('analyzerArn')
+    dArgs['maxResults'] = 25
+
+    nextToken = None
+    incidents = []
+    tmp_last_fetch = last_fetch
+
+    while True:
+
+        if nextToken:
+            dArgs['nextToken'] = nextToken
+        raw_incidents = list_findings_command(dArgs)
```

Comment:
better separate the findings function to a request function and a command function

Reply:
Created a new function `get_findings_command` to use with fetch command.